source/gui/settingsDialogs.py

```diff
@@ -1562,14 +1562,22 @@ def makeSettings(self, settingsSizer):
 			self.shapeList.Disable()
 
 		# Translators: The label for a setting in braille settings to change how long a message stays on the braille display (in seconds).
-		messageTimeoutText = _("Message timeout (sec)")
+		messageTimeoutText = _("Message &timeout (sec)")
 		self.messageTimeoutEdit = sHelper.addLabeledControl(messageTimeoutText, nvdaControls.SelectOnFocusSpinCtrl,
 			min=int(config.conf.getConfigValidationParameter(["braille", "messageTimeout"], "min")),
 			max=int(config.conf.getConfigValidationParameter(["braille", "messageTimeout"], "max")),
 			initial=config.conf["braille"]["messageTimeout"])
+		if self.noMessageTimeoutCheckBox.GetValue():

```

Comment:
This causes an exception, since `self.noMessageTimeoutCheckBox` is not created until line 1575